docs/under-the-hood/replication.md

```diff
@@ -80,13 +80,26 @@ registration.
 
 SYNC mode is the most straightforward replication mode in which the main storage
 thread waits for the response and cannot continue until the response is
-received. That means that the MAIN instance will not commit a transaction until
-all REPLICA instances running in the SYNC mode confirm they have received the
-same transaction.
+received or until a timeout is reached. If the timeout is reached and at least one SYNC REPLICA has not
+sent back a response, then the MAIN instance will then return an error to the user.<br/>
+The error indicates to the user that he should check the status of the REPLICAS
+as there might be a network/hardware issue.
 
-SYNC mode prioritizes data consistency but has no tolerance for any network
-failures because if any of the REPLICATION instances fail, the MAIN instance
-will fail as well.
+The following diagrams express the behavior in case a SYNC REPLICA would not answer within the expected timeout.
+
+#### SYNC REPLICA going down when creating index, uniqueness constraint or existence constraint
+
+![sync-replicas-down-when-creating-index-or-constraints](data/replication/workflow_diagram_data_definition_creation.drawio.png)
+
+
+#### SYNC REPLICA going down when dropping index, uniqueness constraint or existence constraint
+
+![sync-replicas-down-when-dropping-index-or-constraints](data/replication/workflow_diagram_data_definition_dropping.drawio.png)
+
+
+#### SYNC REPLICA going down adding/updating/deleting data
+
+![sync-replicas-down-when-modifying-data](data/replication/workflow_diagram_data_manipulation.drawio.png)
```

Comment:
The (2) in this picture doesn't exist anymore

Reply:
![image](https://user-images.githubusercontent.com/95473291/183595603-ae8db2dd-15fa-474b-8dac-041488b2ce47.png)
Missing a **:** on Uniqueness constraint on properties on 2 pictures

![image](https://user-images.githubusercontent.com/95473291/183596218-58d6c505-8e86-4abc-801e-deb2db9823db.png)
I just need to point out that this Yes is not centered on both pictures... You decide if you'll fix it, I just need to say it :D

Reply:
Both remarks: done